### PR TITLE
Use singular form for per-conversation safety numbers

### PR DESCRIPTION
**Pull request by RiseT**
### First time contributor checklist

- [x] I have read the [README](https://github.com/WhisperSystems/Signal-iOS/blob/master/README.md) and [CONTRIBUTING](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md) documents
- [x] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I'm following the [code, UI and style conventions](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md#code-conventions)
- [x] My commits are rebased on the latest master branch
- [x] My commits are in nice logical chunks
- [x] My contribution is fully baked and is ready to be merged as is
- [ ] I have tested my contribution on these devices:
 * iDevice A, iOS X.Y.Z
 * iDevice B, iOS Z.Y
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in my commit message

- - - - - - - - - -

### Description

Change safety number strings to be singular (approved by Moxie).

Corresponding PR to
WhisperSystems/Signal-Android#5894 (already committed)
WhisperSystems/Signal-Desktop#984

// FREEBIE

### CODE REVIEW

**Comment by RiseT** [comment]
@michaelkirk Ugh. That change to UTF-8 just smashed my first PR. But hey, now there's a diff I can see ;-)

File: Signal/translations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -242,28 +242,28 @@
...
 
 /* No comment provided by engineer. */
 "ERROR_WAS_DETECTED_TITLE" = "Bummer!";
 
 /* action sheet header when re-sending message which failed because of too many attempts */
 "FAILED_SENDING_BECAUSE_RATE_LIMIT" = "Too many failures with this contact. Please try again shortly.";
 
-/* action sheet header when re-sending message which failed because of untrusted identity keys */
-"FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "You must accept their new safety numbers before you will be able to send.";
+/* action sheet header when re-sending message which failed because of an untrusted safety number */
```
Comment (michaelkirk) [inline Signal/translations/en.lproj/Localizable.strings]: These comments are extracted from the source code, so they'll be blown away the next time we generate strings.

Comment (RiseT) [inline Signal/translations/en.lproj/Localizable.strings]: Thanks for the info. I've fixed this for 2 of the 3 changed instructions for translators. The third one (the one above you've commented on) would have to be changed in SignalServiceKit's source code (searched for it for a rather long time...), so I would have to open a separate PR. But that's not *that* important, I guess...?

**Comment by RiseT** [comment]
@michaelkirk Should I squash these?

**Comment by michaelkirk** [comment]
> @michaelkirk Should I squash these?

I will if you don't. Either way.

**Comment by michaelkirk** [comment]
In f29a0fe, thanks!

**Pull request closed without merging**